runtime/src/accounts_index.rs

```diff
@@ -215,19 +215,30 @@ pub struct AccountMapEntryMeta {
     pub dirty: AtomicBool,
     /// 'age' at which this entry should be purged from the cache (implements lru)
     pub age: AtomicU8,
+    /// true if we don't know whether this item exists on disk or not
+    pub disk_unknown: AtomicBool,
```

Comment:
Can this be renamed to `_known` instead of `_unknown`? Then we can avoid the double-negative "not unknown" aka "not not known" checks in the code. I imagine someone eventually will do a `if !disk_unknown` check :)

The helper functions can still be `is_disk_unknown()`, and internally they do the `!disk_known.load()`.

Reply:
yes. will address it later.

Reply:
Rename to `lazy_disk_load` to be more explicitly and clear up the double-negative checks.